### PR TITLE
Flag Case Sensitivity Update

### PR DESCRIPTION
**Pull request by justinwray**
* Flags have now been made case sensitivity.  Previously the case was ignored when checking the correctness of a flag answer.

* Added a Unit Test to ensure the flag was only counted as correct when the case matches.

### CODE REVIEW

File: tests/models/LevelTest.php [hunk 1]
```diff
@@ -178,6 +178,7 @@ public function testAll(): void {
 
   public function testCheckAnswer(): void {
     $this->assertFalse(HH\Asio\join(Level::genCheckAnswer(1, 'no')));
+    $this->assertFlase(HH\Asio\join(Level::genCheckAnswer(1, 'FLAG')));
```
Comment (iangcarroll) [inline tests/models/LevelTest.php]: This should probably be `assertFalse`.

Comment (justinwray) [inline tests/models/LevelTest.php]: Thank you!

**Comment by gsingh93** [comment]
Can you rebase now that the tests are fixed?

**Comment by justinwray** [comment]
@gsingh93 Rebase complete!

**Comment by inche-ali** [comment]
@justinwray 
I think the test failed because, you did not update most of the asertion check value in LevelTest.php
after adding two test data inside seed.xml

for example from line 37 LevelTest.php
`$this->assertEquals(**2**, count($all));`
should be
`$this->assertEquals(**4**, count($all));`

File: tests/models/LevelTest.php [hunk 2]
```diff
@@ -34,7 +34,7 @@ public function testCreateLevel(): void {
 
     $this->assertEquals(2, $id);
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: latest inserted level id should be **4**

File: tests/models/LevelTest.php [hunk 3]
```diff
@@ -68,7 +68,7 @@ public function testCreateFlag(): void {
...
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: latest inserted flag id should be **4**

File: tests/models/LevelTest.php [hunk 4]
```diff
@@ -34,7 +34,7 @@ public function testCreateLevel(): void {
...
     $all = HH\Asio\join(Level::genAllLevels());
-    $this->assertEquals(2, count($all));
+    $this->assertEquals(4, count($all));
     $l = $all[1];
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: index number: $all[**3**]
after added extra 2 test data

File: tests/models/LevelTest.php [hunk 5]
```diff
@@ -68,7 +68,7 @@ public function testCreateFlag(): void {
...
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: index number: $all[**3**]
after added extra 2 test data

File: tests/models/LevelTest.php [hunk 6]
```diff
@@ -34,7 +34,7 @@ public function testCreateLevel(): void {
...
     $this->assertEquals(2, $l->getId());
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: level id should be **4**

File: tests/models/LevelTest.php [hunk 7]
```diff
@@ -68,7 +68,7 @@ public function testCreateFlag(): void {
...
```
Comment (inche-ali) [inline tests/models/LevelTest.php]: level id should be **4**

**Comment by justinwray** [comment]
@inche-ali Thank you!  I am having intermediate test failures (unrelated to this PR), so I missed a few of those.  The LevelTests all completed successfully on commit 22d7a89.

**Comment by gsingh93** [comment]
Thanks!

**Pull request merged**